CMakeLists.txt

```diff
@@ -20,7 +21,7 @@ target_link_libraries(main
   core)
 
 # Add flags.
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++14")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++14 -g -fsanitize=address")
```

Comment:
Yeah.  I think it would be better to make a sanitizer build directory locally by adding the flags when you run CMAKE to set it up - without mentioning it in code anywhere.